downSample.wdl

```diff
@@ -0,0 +1,387 @@
+version 1.0
+
+struct fastqPair {
+    File fastq1
+    File fastq2
+}
+
+struct bamFile {
+  File bam
+  File bamIndex
+}
+
+workflow downSample {
+    input {
+        fastqPair? inputFastq
+        bamFile? inputBam
+        String? refFasta
```

Comment:
It accepts reference fasta, not a reference id. Do we want to leave it like this or lock it so that ref and other ref-specific resources specified inside the wdl, like in the other workflows?

Reply:
refFasta is only used in check coverage after down sample bam, and there is no other ref-specific resources. but noticing in the olive we alwasy use reference module to locate the refFasta, so I changed to use reference as input. Please see new commit. Thanks!